key_lists/debug.yml

```diff
@@ -1,7 +1,9 @@
 ---
 # Quick shortlist of keys to help debug analysis code.
 keys:
-    Temperature_debug: True
+    MA_Totalairseafluxco2: True
```

Comment:
this is why the test is failing, do you need this?

Reply:
I mean, the test is failing smartly, it needs that yml file :grin:

Reply:
It's failing because we're not supposed to be using `debug.yml` in the tests. We created a new file for that, `integration_test.yml`. `debug.yml` is not for testing on github. it's for debugging!

Reply:
Having said that, I can't see where the test calls it!

Reply:
fixed it with my last commit, it was the input yml that was still using debug as suite - thought I'd fixed that in the other PR

Reply:
bam! green tests, dude :green_circle: